fontforge/nowakowskittfinstr.c

```diff
@@ -692,7 +692,7 @@ static void init_cvt(GlobalInstrCt *gic) {
     cvtsize += gic->stemsnapvcnt;
     cvtsize += gic->bluecnt * 2; /* possible family blues */
 
-    cvt = calloc(cvtsize, cvtsize * sizeof(int16));
```

Comment:
Given the context, this is actually correct; I wouldn't follow the static analyser on cases like these.

Reply:
So revert, or do `sizeof(char)` to quiet the warning?

Reply:
Reverted.